nbviewer/providers/gist/handlers.py

```diff
@@ -23,18 +23,27 @@
 
 from .. import _load_handler_from_location
 
-
 class GistClientMixin(GithubClientMixin):
+    """
+    provider_label: str
+        Text to to apply to the navbar icon linking to the provider
+    provider_icon: str
+        CSS classname to apply to the navbar icon linking to the provider
+    executor_label: str, optional
+        Text to apply to the navbar icon linking to the execution service
+    executor_icon: str, optional
+        CSS classname to apply to the navbar icon linking to the execution service
+    """
```

Comment:
Are these instance attributes? Constructor arguments? Let's add a note to the beginning of the docstring explaining what these are.